frontend/src/models/api/Property.ts

```diff
@@ -0,0 +1,7 @@
+import { Api_ConcurrentVersion } from './ConcurrentVersion';
+
+export interface Api_Property extends Api_ConcurrentVersion {
+  id?: string;
```

Comment:
Looks like you are trimming out some content from 2907 with this.
2907 supports:
pid, pin, plan number, latitude, and longitude.
it will eventually support address and legal description when we get bc assessment and the full attributed parcel map working.